src/tests/ftest/control/dmg_network_scan.yaml

```
@@ -0,0 +1,13 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+  test_servers:
+    - boro-A
```

Recently we changed the format of the host placeholders.  To confirm with the new standard this should be:
```suggestion
    - server-A
```

---

Will change.